evm/src/NttManager/ManagerBase.sol

```diff
@@ -423,7 +423,7 @@ abstract contract ManagerBase is
         _setTransceiverAttestedToMessage(digest, _getTransceiverInfosStorage()[transceiver].index);
 
         emit MessageAttestedTo(
-            digest, transceiver, _getTransceiverInfosStorage()[transceiver].index
+            chainId, digest, transceiver, _getTransceiverInfosStorage()[transceiver].index
```

Comment:
Isn't `chainId` the current chain, not the source chain id of the message?

Reply:
You're right, `sourceChainId` is not in the scope of `_setTransceiverAttestedToMessage` function so will pass it in as a param from the calling function.